### DISCUSSION TITLE
Insights - Allow Dashboards sorting

### DISCUSSION THREAD

**Discussion by igiornotigi** [directus/directus#8903] [Feature Requests]
Hello everybody, I would like to suggest adding the ability to custom sort (drag and drop) the created dashbords. Currently they are automatically sorted in alphabetical order.
Thanks

![image](https://user-images.githubusercontent.com/57590952/137728837-7bebfc4b-fd30-441c-9333-7e77639d41cf.png)

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
